proto/poktroll/migration/morse_onchain.proto

```diff
@@ -20,14 +20,17 @@ message MorseAccountState {
 // account which is claimable as part of the Morse -> Shannon migration.
 // They are intended to be created during MorseAccountState import (see: MsgImportMorseClaimableAccount).
 message MorseClaimableAccount {
-  // A hex-encoded representation of the address corresponding to a Morse application's ed25519 public key.
-  bytes address = 1 [(gogoproto.jsontag) = "address", (gogoproto.casttype) = "github.com/cometbft/cometbft/crypto.Address"];
+  // The bech32-encoded address of the Shannon account to which the claimed balance will be minted.
```

Comment:
#PUC that this will be empty until the account is claimed.

Reply:
The `(gogoproto.nullable)` option only applies to non-primitive types.